packages/format-library/src/bold/index.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { Fragment } from '@wordpress/element';
+import { ToolbarButton } from '@wordpress/components';
+import { toggleFormat } from '@wordpress/rich-text';
+
+export const bold = {
+	name: 'core/bold',
+	title: __( 'Bold' ),
+	match: {
+		tagName: 'strong',
+	},
+	edit( { isActive, value, onChange, FillToolbarSlot, Shortcut } ) {
+		const onToggle = () => onChange( toggleFormat( value, { type: 'core/bold' } ) );
```

Comment:
Should we move `core/bold` to constant to have this value defined in one place?

Reply:
Yes. This is a still a WIP :)